internal/executor/job/testfixtures.go

```diff
@@ -0,0 +1,84 @@
+package job
```

Comment:
I'd either name this `testfixtures_test.go` or move it into a special `testfixtures` package.  As it stands right now, I think this will be compiled into our prod binary.